packages/docs/src/pages/en/features/sass-variables.md

```diff
@@ -29,7 +29,30 @@ When you run yarn serve, the vuetify-cli-plugin will automatically hoist the glo
 
 ## Nuxt install
 
-Coming soon. If you are interested in collaborating on this section, please reach out to _johnleider_ in the [community](https://community.vuetifyjs.com).
+This section assumes that you have already followed our Nuxt guide on the [Quick start](/getting-started/installation/#nuxt-install) page. The Nuxt Vuetify module exposes a `vuetify` property where you can configure the module. Ensure that you enable the `treeShake` option first. This option will use [vuetify-loader](https://github.com/vuetifyjs/vuetify-loader) to enable automatic [treeshaking](/features/treeshaking). This is required for custom SASS variables to work. Then provide the `customVariables` file path option to customize SASS variables.
+
+```js
+// nuxt.config.js
+
+export default {
+  vuetify: {
+    customVariables: ['~/assets/variables.scss'],
+    treeShake: true
+  },
+}
+```
+
+```scss
+// assets/variables.scss
+
+// Variables you want to modify
+$btn-border-radius: 0px;
+
+// Modifying a SASS map variable
+$material-light: ( cards: blue );
+
+@import '~vuetify/src/styles/styles.sass';
```

Comment:
This import isn't needed

Reply:
It looks like @vuetify/nuxt docs need to be updated